README.md

```diff
@@ -0,0 +1,5 @@
+1. Absolute vs Relative Paths
+2. Using CRUD to update a database
+3. Regex.
+4. Animating svg files with css/js
+5. Media queries for designing mobile web pages
```

Comment:
This is very interesting especially since phones keep being released with higher pixel rates